app/decorators/mod_action_decorator.rb

```diff
@@ -19,8 +19,6 @@ def format_description
       ### Pools ###
     when "pool_delete"
       "Deleted pool ##{vals['pool_id']} (named #{vals['pool_name']}) by #{user}"
-    when "pool_undelete"
```

Comment:
Better to leave existing mod actions as there are often existing entries for them. Without this decorator they display as errors.

Reply:
Yeah, but there is only 1 so I thought it wasn't  worth it. I can add it back if you like though